ui/components/app/modals/import-token-exit-modal/import-token-exit-modal.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import * as actions from '../../../../store/actions';
+import { DEFAULT_ROUTE } from '../../../../helpers/constants/routes';
+import {
+  Text,
+  ButtonSecondary,
+  ButtonPrimary,
+  Box,
+} from '../../../component-library';
+import {
+  TextAlign,
+  FontWeight,
+  TextVariant,
+  Size,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+export default function ImportTokensExitModal() {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const later = (delay) => {
+    return new Promise(function (resolve) {
+      setTimeout(resolve, delay);
+    });
+  };
+
+  const hideModal = () => dispatch(actions.hideModal());
+
+  const forceHideModal = () => {
+    later(0).then(() => {
+      dispatch(actions.clearPendingTokens());
+      dispatch(actions.hideModal());
+    });
+  };
+
+  return (
+    <div className="exit-token-confirmation">
```

Comment:
This should be wrapped inside Modal and div should be replaced by Box, that way we won't have to deal with the extra css and we can use Box utiltity props

Reply:
updated 🙏